oci-unit-tests/postgresql_test.sh

```diff
@@ -8,12 +8,22 @@
 #  setUp() - run before each test
 #  tearDown() - run after each test
 
+DOCKER_NETWORK_NAME=postgresql_test
+
+oneTimeSetUp() {
+    docker network create $DOCKER_NETWORK_NAME
+}
+
 setUp() {
     password=$(dd if=/dev/urandom bs=1 count=16 2>/dev/null | md5sum | head -c 16)
     id=$$
     image="squeakywheel/postgres:edge"
 }
 
+oneTimeTearDown() {
+    docker network rm $DOCKER_NETWORK_NAME
+}
+
```

Comment:
Have you checked that these setup and teardown work as expected? For example, induce a failure in a test later on, and check that the network is removed.

Reply:
Yes, they work as expected.  If any test fails, the network still gets removed.